sphinx/ext/autosummary/__init__.py

```diff
@@ -298,6 +298,15 @@ def import_by_name(self, name: str, prefixes: List[str]) -> Tuple[str, Any, Any,
 
                 raise exc  # re-raise ImportError if instance attribute not found
 
+    def create_documenter(self, app: Sphinx, obj: Any,
+                          parent: Any, name: str) -> "Type[Documenter]":
+        """Get an autodoc.Documenter class suitable for documenting the given
+        object.
+
+        Wraps get_documenter and is meant as a hook for extensions.
+        """
+        return get_documenter(app, obj, parent)
```

Comment:
Sorry for my lack of words. What I meant is this implementation
```suggestion
        doccls = get_documenter(app, obj, parent)
        return doccls(self.bridge, name)
```

Reply:
done